README.md

```diff
@@ -1,37 +1,134 @@
+  - Integration with WhatsApp using its [click to chat](https://faq.whatsapp.com/5913398998672934) functionality.
+- A `package.json` file with your app's dependencies.
+
+## Step 2 | Test the app
+
+### Install the Wix Stores app
+
+This app makes calls to the Wix eCommerce [Orders API](https://dev.wix.com/docs/sdk/backend-modules/ecom/orders/setup) to retrieve the latest orders made in the site’s store. Before you test this app on your development site, install the [Wix Stores app](https://www.wix.com/app-market/wix-stores) to [add the Wix eCommerce platform’s functionality](https://dev.wix.com/docs/rest/business-solutions/e-commerce/wix-e-commerce-platform-handbook/get-started/installation) to your site.
+
+### Set up a local development environment
+
+The app creation process installs the app on your chosen development site. However, you won’t see the app extensions on your site until you build the app and create a version. To test the app during development, set up a local development environment using the following command:
+
+```tsx
+npm run dev
 ```
+
+The development environment is set up for hot reloading, so any changes you make to your code will be reflected in the browser.
+
+### Testing steps
+
+1. After setting up the local development environment, the CLI will prompt you with a menu. Press **D** to select dashboard, then press **1**. This opens your development sites' dashboard homepage.
+2. Navigate to the **Sales > Orders** page and click on an order to open its specific order page.
+
+  [screenshot]()
+
+3. Click the **More actions** button, and then select **Post-Order Messaging** from the menu.
+
+
+  [screenshot]()
+
+4. Select a product or products to sell, then click **Send Message**. This opens WhatsApp Web to a newly created message ready to send to the customer.
+
+
+  [screenshot]()
+
+## Extend and customize the app 
+
+The template is designed for easy customization and extension. Here are some suggested entry points where you can add your own custom logic or functionality:
+
+### Customize the dashboard menu plugin
+
+You can customize the name and icon of the dashboard menu plugin, and what happens when it's clicked.
+
+Development entry point: [`template/src/dashboard/menu-plugins/my-plugin/plugin.json`](./template/src/dashboard/menu-plugins/my-plugin/plugin.json)
+
+You can also replace this plugin with a plugin for another of the [Order page's plugin slots](https://dev.wix.com/docs/build-apps/develop-your-app/extensions/dashboard-extensions/dashboard-plugins/supported-wix-app-dashboard-pages/wix-e-commerce/wix-e-commerce-order-page#slot-4-more-actions-menu).
+
+### Customize the dashboard modal
+
+Customize the dashboard modal to fit your specific needs, whether it's updating the UI or changing the functionality.
+
+Development entry point: [`template/src/dashboard/modals/sell-message-modal/modal.tsx`](./template/src/dashboard/modals/sell-message-modal/modal.tsx)
+
+### Customize the WhatsApp integration
```

Comment:
Maybe the heading should be "Customize the messaging integration", since the idea is that a dev can implement here any integration with any service they like?